clang/lib/DPCT/Rules.h

```diff
@@ -21,37 +21,37 @@ enum RulePriority { Takeover, Default, Fallback };
 
 struct TypeNameRule {
   std::string NewName;
-  clang::dpct::HelperFeatureEnum RequestFeature;
+  bool NeedDpctDeviceExtFlag;
   RulePriority Priority;
   std::vector<std::string> Includes;
   TypeNameRule(std::string Name)
-      : NewName(Name),
-        RequestFeature(clang::dpct::HelperFeatureEnum::no_feature_helper),
+      : NewName(Name), NeedDpctDeviceExtFlag(false),
```

Comment:
the default value is `true` may be more safety.

Reply:
Maybe follow the original logic in this PR is better?